src/tests/end-to-end/common/page-controllers/target-page.ts

```diff
@@ -38,4 +39,11 @@ export class TargetPage extends Page {
         const shadowRoot = await this.getShadowRoot();
         return await formatChildElementForSnapshot(shadowRoot, '#insights-shadow-container');
     }
+
+    public async waitForVisualizationBoxesToDisappear(): Promise<void> {
+        if (document.querySelector(TargetPageInjectedComponentSelectors.insightsRootContainer)) {
```

Comment:
>document [](start = 12, length = 8)

This is the jsdom document provided by node, not the document of the target page